tdrs-frontend/src/components/Header/Header.jsx

```diff
@@ -41,17 +42,14 @@ function HeaderComp() {
               <img src={closeIcon} alt="close" />
             </button>
             <ul className="usa-nav__primary usa-accordion">
-              <li className="usa-nav__primary-item">
-                <a
-                  href="/"
-                  key="welcome"
-                  className={`usa-nav__link ${
-                    pathname === '/edit-profile' ? 'usa-current' : ''
-                  }`}
-                >
-                  <span>Welcome</span>
-                </a>
-              </li>
+              <NavItem pathname={pathname} tabTitle="Welcome" href="/welcome" />
+              <NavItem pathname={pathname} tabTitle="Reports" href="/reports" />
+              <NavItem
+                pathname={pathname}
+                tabTitle="Profile"
+                href="/edit-profile"
+              />
+              <NavItem pathname={pathname} tabTitle="Admin" href="/admin" />
```

Comment:
This isn't a blocker, but just thinking ahead ...

@carltonsmith @hilvitzs Will this "Admin" tab will route to the Django Admin page? Assuming eventually we'll need to scope this so that only the superuser sees a link to "Admin". Fine for now, just noting/thinking out loud.

Reply:
Honestly, I was just matching the designs on this one so I'm sure that we'll have a discussion around which tabs are visible for each role. I'll make sure and keep this on our radar moving forward.

Reply:
I was assuming this would be OFA admin so good to discuss

Reply:
I don't think there should be a link to this since the access is so limited